xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -1433,6 +1425,29 @@ void CXBMCApp::onReceive(CJNIIntent intent)
   }
 }
 
+void CXBMCApp::OnSleep()
+{
+  CLog::Log(LOGINFO, "CXBMCApp::OnSleep");
+  IPowerSyscall* syscall = CServiceBroker::GetPowerManager().GetPowerSyscall();
+  if (syscall)
+    static_cast<CAndroidPowerSyscall*>(syscall)->SetSuspended();
+}
+
+void CXBMCApp::OnWakeup()
+{
+  CLog::Log(LOGINFO, "CXBMCApp::OnWakeup");
```

Comment:
```suggestion
  CLog::Log(LOGDEBUG, "CXBMCApp::OnWakeup");
```

Same: is a bit redundant of "Got device wakeup intent", keep in debug only